<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="pageSupression">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>681</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTableWidget" name="tableEmployer8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>561</width>
        <height>381</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>tableEmployer</string>
       </property>
      </row>
     </widget>
     <widget class="QPushButton" name="btnActualiser">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>350</y>
        <width>131</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Actualiser la liste</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableEmployer">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>100</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>60</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>comboEmployes</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="SupprimerEmployer">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>400</y>
        <width>151</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>SupprimerEmployer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>32</x>
        <y>60</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>combo Box  :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>170</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Label  :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>180</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>  labelMessage</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
    <widget class="QWidget" name="page_4">
     <widget class="QLineEdit" name="lineEdit_nom">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>50</y>
        <width>121</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>  lineEditNom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prenom">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>130</y>
        <width>131</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>  lineEditPrenom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_salaire">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>190</y>
        <width>131</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>  lineEditSalaire</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_poste">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>320</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>  lineEditPoste</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>250</y>
        <width>171</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>comboDepartement</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>330</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> Poste  :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>250</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Département  :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>  Nom   :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> Prénom  :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>190</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Salaire  :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AjouterEmployer">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>410</y>
        <width>141</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>AjouterEmployer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>50</y>
        <width>121</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>  lineEditNom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_mail">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>120</y>
        <width>121</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>  lineEditNom</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
